components/public/sites/StoreListItem.js

```diff
@@ -49,7 +49,7 @@ const StoreListItem = ({ toggle, setUserStore, site }) => {
       </Row>
     );
   } else {
-    return;
+    return <></>;
```

Comment:
are these open and close tags needed?

Reply:
Yes - it needs to return some type of html and `<></>` is shorthand for `<div></div>`